### PR TITLE
doc improvement: board_target: align formatting

### PR DESCRIPTION
**Pull request by greg-fer**
Applied consistent formatting to west build commands that mention board_targets.

### CODE REVIEW

**Review by tomchy — APPROVED** [review]

File: doc/nrf/config_and_build/bootloaders/bootloader_downgrade_protection.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can compile your application with this feature as follows:
 .. parsed-literal::
```
Comment (MarekPieta) [inline doc/nrf/config_and_build/bootloaders/bootloader_downgrade_protection.rst:40]: In commit message: `doc: board_target:` ?

File: doc/nrf/device_guides/nrf91/nrf91_snippet.rst [hunk 2]
```diff
@@ -30,7 +30,7 @@ The following board targets have support for this snippet:
 * ``nrf9160dk/nrf9160/ns``
 * ``nrf9131ek/nrf9131/ns``
 
-To enable modem traces with the flash backend, use the following command:
+To enable modem traces with the flash backend, use the following command, where *board_target* corresponds to your development kit board target:
```
Comment (MarekPieta) [inline doc/nrf/device_guides/nrf91/nrf91_snippet.rst:33]: ... and `<image_name>` to your application image name ?

File: tests/tfm/tfm_psa_test/README.rst [hunk 3]
```diff
@@ -50,19 +50,21 @@ Building and running
...
 To provision the device, build and flash the provisioning image sample.
-Then run the PSA test suite with the following command:
+Then run the PSA test suite with the following command, where *board_target* is your board target name:
 
-.. code-block:: console
+.. parsed-literal::
+   :class: highlight
 
-    west build -b <build_target> nrf/tests/tfm/tfm_psa_test -- -DCONFIG_TFM_PSA_TEST_INITIAL_ATTESTATION=y -DCONFIG_TFM_PARTITION_INITIAL_ATTESTATION=y -DCONFIG_TFM_NRF_PROVISIONING=y
+    west build -b *build_target* nrf/tests/tfm/tfm_psa_test -- -DCONFIG_TFM_PSA_TEST_INITIAL_ATTESTATION=y -DCONFIG_TFM_PARTITION_INITIAL_ATTESTATION=y -DCONFIG_TFM_NRF_PROVISIONING=y
```
Comment (MarekPieta) [inline tests/tfm/tfm_psa_test/README.rst:67]: board_target?

**Review by MarekPieta — APPROVED** [review]

**Review by jayteemo — APPROVED** [review]

**Review by plskeggs — APPROVED** [review]

File: doc/nrf/device_guides/nrf91/nrf91_snippet.rst [hunk 4]
```diff
@@ -30,7 +30,7 @@ The following board targets have support for this snippet:
 * ``nrf9160dk/nrf9160/ns``
 * ``nrf9131ek/nrf9131/ns``
 
-To enable modem traces with the flash backend, use the following command:
+To enable modem traces with the flash backend, use the following command, where *board_target* corresponds to your development kit board target and `<image_name>` to your application image name:
```
Comment (divipillai) [inline doc/nrf/device_guides/nrf91/nrf91_snippet.rst:33]: board target -> board_target on line no:38.
Also on line no: 89.

File: doc/nrf/test_and_optimize/test_framework/running_unit_tests.rst [hunk 5]
```diff
@@ -51,9 +51,10 @@ To generate the test project and run the unit tests, locate the directory with t
 
    .. group-tab:: west
 
-      .. code-block:: console
+      .. parsed-literal::
```
Comment (divipillai) [inline doc/nrf/test_and_optimize/test_framework/running_unit_tests.rst:54]: Shouldn't the line <board_target> on no:31 and 45 also change to *board_target*?

**Review by umapraseeda — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Pull request merged**